Update __init__.py

TonyDiana opened the pull request:
Hello. This simple change allows it to be used within the solution, without having to install it (my case). By the way, I've seen this

```python
from alejandria.STD import Singleton, SingletonArgs, ThreadSingletonArgs, ThreadSingleton


class pepe1(object, metaclass=Singleton):
    def __init__(self, casa="mia") -> None:
        self.casa = casa
a = pepe1(casa="1")
b = pepe1(casa="12")
c = pepe1()

# --- OK
print(f"Singleton // {a.casa} vs {b.casa} vs {c.casa}")


class pepe2(object, metaclass=SingletonArgs):
    def __init__(self, casa="mia") -> None:
        self.casa = casa


a = pepe2(casa="1")
b = pepe2(casa="12")
c = pepe2()

# --- :( NO WORK
print(f"SingletonArgs // {a.casa} vs {b.casa} vs {c.casa}")


class pepe3(object, metaclass=ThreadSingletonArgs):
    def __init__(self, casa="mia") -> None:
        self.casa = casa


a = pepe3(casa="1")
b = pepe3(casa="12")
c = pepe3()

# --- :( NO WORK
print(f"ThreadSingletonArgs // {a.casa} vs {b.casa} vs {c.casa}")


class pepe4(object, metaclass=ThreadSingleton):
    def __init__(self, casa="mia") -> None:
        self.casa = casa


a = pepe4(casa="1")
b = pepe4(casa="12")
c = pepe4()

# --- OK
print(f"ThreadSingleton // {a.casa} vs {b.casa} vs {c.casa}")
```
```
Singleton // 1 vs 1 vs 1
SingletonArgs // 1 vs 12 vs mia
ThreadSingletonArgs // 1 vs 12 vs mia
ThreadSingleton // 1 vs 1 vs 1
```

TonyDiana committed:
Update __init__.py

Hello. This simple change allows it to be used within the solution, without having to install it (my case). By the way, I've seen this

from alejandria.STD import Singleton, SingletonArgs, ThreadSingletonArgs, ThreadSingleton


class pepe1(object, metaclass=Singleton):
    def __init__(self, casa="mia") -> None:
        self.casa = casa
a = pepe1(casa="1")
b = pepe1(casa="12")
c = pepe1()

# --- OK
print(f"Singleton // {a.casa} vs {b.casa} vs {c.casa}")


class pepe2(object, metaclass=SingletonArgs):
    def __init__(self, casa="mia") -> None:
        self.casa = casa


a = pepe2(casa="1")
b = pepe2(casa="12")
c = pepe2()

# --- :( NO WORK
print(f"SingletonArgs // {a.casa} vs {b.casa} vs {c.casa}")


class pepe3(object, metaclass=ThreadSingletonArgs):
    def __init__(self, casa="mia") -> None:
        self.casa = casa


a = pepe3(casa="1")
b = pepe3(casa="12")
c = pepe3()

# --- :( NO WORK
print(f"ThreadSingletonArgs // {a.casa} vs {b.casa} vs {c.casa}")


class pepe4(object, metaclass=ThreadSingleton):
    def __init__(self, casa="mia") -> None:
        self.casa = casa


a = pepe4(casa="1")
b = pepe4(casa="12")
c = pepe4()

# --- OK
print(f"ThreadSingleton // {a.casa} vs {b.casa} vs {c.casa}")

Singleton // 1 vs 1 vs 1
SingletonArgs // 1 vs 12 vs mia
ThreadSingletonArgs // 1 vs 12 vs mia
ThreadSingleton // 1 vs 1 vs 1

mammo0 commented:
Hello,

**First thing**: Please use code blocks in markdown, when you write code samples. This makes the text more readable. This time I edited the text for you ;)

**Second**: I will accept your PR. I use relative imports everywhere else, so it will be consistent now.

**Third**:
> By the way, I've seen this

The next time please open an issue if you found a bug. Do not put all things together.

Nevertheless I had a look at your code sample. But I don't get the point what's not working...
For example I take your second sample:
```python
class pepe2(object, metaclass=SingletonArgs):
    def __init__(self, casa="mia") -> None:
        self.casa = casa


a = pepe2(casa="1")
b = pepe2(casa="12")
c = pepe2()

print(f"SingletonArgs // {a.casa} vs {b.casa} vs {c.casa}")
```

Output:
```
SingletonArgs // 1 vs 12 vs mia
```

The output is the intended behavior. Please have a look at the README: https://github.com/mammo0/py-simple-singleton#difference-between-singleton-and-singletonargs

There this behavior is explained.

TonyDiana commented:
Sorry, I'm still learning to use github, hehehe, yes, I misunderstood how the library works. Thanks for everything

TonyDiana commented:
I already found out what was my reason for haste and fright. I'm using this singleton on an object that operates through a multiprocess, not a multithread. This object allows sharing memory between processes, through the proxy. It stopped working when I changed the library, but what I didn't see is that, when I changed it, I put my finger in and commented a line that I shouldn't comment, as we say in Spain, it's a layer 8 problem, or as they say in English, PEBKAC ERROR :D

I want to confirm that the operation of this stupendous multithreaded utility is brilliant.

mammo0 commented:
Thank you for the positive feedback :)

TonyDiana commented:
Hello, I need to contact you, I don't know how to do it more privately by this means

mammo0 commented:
I wrote you an e-mail.
